### PR TITLE
[PT] Update enum functionalities for Elixir 1.4 (#696).

### PR DESCRIPTION
**Pull request by anderkonzen**
Checklist for this update is in #701.

### CODE REVIEW

**Comment by ybur-yug** [comment]
Thanks for PR-ing this to the improvement/1.4 branch! Have someone else peek over it and we can merge as soon as theres a 👍  :) @anderkonzen

**Comment by nscyclone** [comment]
@pragmaticivan, @ruan-brandao, could you review this, please?

File: pt/lessons/basics/enum.md [hunk 1]
```diff
@@ -56,6 +56,15 @@ iex> Enum.chunk_by(["one", "two", "three", "four", "five"], fn(x) -> String.leng
 [["one", "two"], ["three"], ["four", "five"]]
 ```
 
+### map_every
+
+Algumas vezes quebrar uma coleção não é o suficiente para exatamente o que você precisa. Se este é o caso, `map_every/3` pode ser muito útil para tratar apenas itens específicos se sua coleção possui uma ordenação que possa ser necessária ou útil:
```
Comment (ruan-brandao) [inline pt/lessons/basics/enum.md]: I think "para _fazer_ exatamente o que você precisa" would be better.

**Comment by ruan-brandao** [comment]
Just left a review commwnt. Other than that, great work :+1:

**Comment by anderkonzen** [comment]
Great, thanks @ruan-brandao!

**Comment by ybur-yug** [comment]
Thanks @ruan-brandao! Merging :)

**Pull request merged**